backend/sites/src/app/entities/siteProfiles.entity.ts

```diff
@@ -19,19 +20,34 @@ import { ChangeAuditEntity } from './changeAuditEntity';
 @Index('site_profiles_pkey', ['dateCompleted', 'siteId'], { unique: true })
 @Index('sprof_rwm_site_partic', ['rwmParticId'], {})
 @Index('sprof_site_reg_site_partic', ['siteRegParticId'], {})
+@Index('site_profiles_id', ['id'], {})
 @Entity('site_profiles')
 export class SiteProfiles extends ChangeAuditEntity {
+  // ADDED THIS PARIMARY KEY COLUMN AND REMOVE COMPOSITE PRIMARY KEY COLUMN
+  // FOR CRUD OPERATION
   @Field()
-  @Column('bigint', { primary: true, name: 'site_id' })
+  @PrimaryGeneratedColumn('uuid', { name: 'id' })
+  id: string;
+
+  @Field()
+  @Column('bigint', { name: 'site_id' })
   siteId: string;
 
   @Field()
-  @Column('timestamp without time zone', {
-    primary: true,
-    name: 'date_completed',
-  })
+  @Column('timestamp without time zone', { name: 'date_completed' })
   dateCompleted: Date;
 
+  // @Field()
+  // @Column('bigint', { primary: true, name: 'site_id' })
+  // siteId: string;
+
+  // @Field()
```

Comment:
Is this commented for future reference ? if not please remove the commented code.

Reply:
yes for future